resources/langs/nheko_de.ts

```diff
@@ -6,89 +6,89 @@
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="32"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>ABBRECHEN</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="43"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>Titel</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="46"/>
         <source>Topic</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Thema/Beschreibung</translation>
```

Comment:
This needs to be discussed which one fits best in here. I guess @krombel might have some ideas on this

Reply:
In Riot this is called "Thema". And that would be a more direct translation.
"Beschreibung" is "description". As this field is mostly used for that I could agree with that translation but I would prefer to have that consistent to "upstream"